Makefile.in

```diff
@@ -52,8 +52,8 @@ DISTDIR=$(top_builddir)/$(package)
 FILES:=Makefile.in configure configure.ac dtls_config.h.in \
   Makefile.tinydtls $(SOURCES) $(HEADERS)
 LIB:=libtinydtls
-LIBS:=$(LIB).a $(LIB).so
-LDFLAGS:=@LIBS@
+LIBS:=$(LIB).a $(LIB).so @LIBS@
```

Comment:
I think that `@LIBS@` must be in `LDFLAGS`. It is set by `configure` on platforms that have `-liblnsl -libsocket` (e.g., SunOS). The Makefile variable `LIBS`, on the other hand, contains the make targets governed by this Makefile.

Reply:
OK, I'll update the PR accordingly